networking/multiple_networks/understanding-multiple-networks.adoc

```diff
@@ -67,4 +67,13 @@ networks in your cluster:
 
 * *TAP*: xref:../../networking/multiple_networks/secondary_networks/creating-secondary-nwt-other-cni.adoc#nw-multus-tap-object_configuring-additional-network-cni[Configure a TAP-based additional network] to create a tap device inside the container namespace. A TAP device enables user space programs to send and receive network packets.
 
-* *SR-IOV*: xref:../../networking/hardware_networks/about-sriov.adoc#about-sriov[Configure an SR-IOV based additional network] to allow pods to attach to a virtual function (VF) interface on SR-IOV capable hardware on the host system.
+ * *SR-IOV*: xref:../../networking/hardware_networks/about-sriov.adoc#about-sriov[Configure an SR-IOV based additional network] to allow pods to attach to a virtual function (VF) interface on SR-IOV capable hardware on the host system.
+
+[id="udn-support-matrix_{context}"]
+== Support matrix for UserDefinedNetworks and secondary networks {product-title}
+
+include::modules/nw-udn-support-matrix-primary-secondary.adoc[leveloffset=+1]
```

Comment:
If the above is not a leftover, this should likely be bumped to `+2` - right now it's the same level as the one above it